pom.xml

```diff
@@ -72,6 +86,12 @@
       <groupId>uk.gov.companieshouse</groupId>
       <artifactId>structured-logging</artifactId>
       <version>${structured-logging.version}</version>
+      <exclusions>
```

Comment:
Why the exclude of logback? Won't this force the app to use log4j whereas it was probably using logback before (depending on arcane slf4j rules of deciding which framework to use based on what's on the classpath)?